doc/topics/windows/windows-package-manager.rst

```diff
@@ -1221,47 +1194,33 @@ ready for use.
 Troubleshooting
 ***************
 
-My software installs correctly but pkg.installed says it failed
+My software installs correctly but `pkg.installed says it failed
```

Comment:
**Question:** Do you need a closing back-tick after pkg.installed? Also, for rST I think it should be double back-ticks?

Reply:
Updated